README.md

```diff
@@ -33,7 +37,7 @@ so that we can ensure we are solving the right problems in the right way.
 - Comment on existing issues to give your feedback on how they should be fixed/implemented
 - Contribute a bug fix or feature implementation by submitting a pull request
 - Contribute more unit tests for feature areas that lack good coverage
-- Review the pull requests that others submit to ensure they follow [established guidelines]
+ Review the pull requests that others submit to ensure they follow [established guidelines]
```

Comment:
Why did you remove the - ?

Reply:
i think it had 2, i only meant to delete 1
BTW the markdownlint in vs code thinks our repo favors asterisks over hyphens for lists. Is that in a config file somewhere?
![image](https://user-images.githubusercontent.com/2224906/91894254-66de8d00-ec63-11ea-9422-637d90248905.png)